app-android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.22.1)
```

Comment:
did it not work out to avoid cmake/ninja?

Reply:
I mainly just didn't want to change what was already working.

I don't know enough about Android development to make a well-informed decision towards one build system over another, other than what had been recommended to me.

Reply:
I also don't know what works best, my only concern is that we would need more tooling installed